ledger/light/proc/set_request.go

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package proc
+
+import (
+	"context"
+
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/bus"
+	"github.com/insolar/insolar/insolar/flow"
+	"github.com/insolar/insolar/insolar/payload"
+	"github.com/insolar/insolar/insolar/record"
+	"github.com/insolar/insolar/ledger/light/hot"
+	"github.com/insolar/insolar/ledger/light/recentstorage"
+	"github.com/insolar/insolar/ledger/object"
+	"github.com/pkg/errors"
+)
+
+type SetRequest struct {
+	message   payload.Meta
+	request   record.Virtual
+	requestID insolar.ID
+	jetID     insolar.JetID
+
+	dep struct {
+		writer        hot.WriteAccessor
+		records       object.RecordModifier
+		recentStorage recentstorage.Provider
+		pendings      object.PendingModifier
+		pendingsLimit int
```

Comment:
This is not a dependency. Lets just hard-code it as const.

Reply:
Done. It's a const now. Into recentstorage.Provider